### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by stephamon**
fIx minor typo

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-5a83e042e4/how-to/)

File: content/cloud-servers/install-epel-and-additional-repositories-on-centos-and-red-hat.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ repository provides useful software packages that are not included in
 the official CentOS or Red Hat repositories.
 
 Instructions are also included for installing the [IUS Community Project](https://ius.io/). Whereas EPEL provides
-only software that is *not* in the official CentOS and Red Hat official repositories, IUS provides newer versions of software (like MySQL and PHP) that exists in the official repositories.
+only software that is *not* in the official CentOS and Red Hat official repositories, IUS provides newer versions of software (like MySQL and PHP) that exist in the official repositories.
```
Comment (catlook) [inline content/cloud-servers/install-epel-and-additional-repositories-on-centos-and-red-hat.md]: does "official" need to be in sentence twice?

Comment (stephamon) [reply content/cloud-servers/install-epel-and-additional-repositories-on-centos-and-red-hat.md]: Haha probably not

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-d4d45cd072/how-to/).

**Pull request merged**